src/amb_kf.c

```diff
@@ -867,7 +890,7 @@ void nkf_state_inclusion(nkf_t *kf,
   double new_cov[new_state_dim * new_state_dim];
   memset(new_cov, 0, new_state_dim * new_state_dim * sizeof(double));
   double new_mean[new_state_dim];
-  memset(new_mean, 0, new_state_dim * sizeof(double));
+  memcpy(new_mean, estimates, new_state_dim * sizeof(double));
```

Comment:
Maybe a comment to say that you are initializing the means here with the ambiguity estimates, and the means for the non new satellites will be overwritten with their existing values below.